distribution/docs/src/main/jdocs/content/_plugins/point-of-contact-policy-plugin-contents.adoc

```diff
@@ -0,0 +1,20 @@
+:type: plugin
+:status: published
+:title: Point of Contact Policy Plugin
+:link: _point_of_contact_policy_plugin
+:plugintypes: policy
+:summary: Adds a policy if Point of Contact is updated.
+
+The Point of Contact Policy Plugin adds a policy if the Point-of-Contact field is changed during pre-update, thus rejecting the entire update operation.
+
+===== Related Components to Point of Contact Policy Plugin
+
+None.
```

Comment:
❓ This plugin is closely related to the `PointOfContactUpdatePlugin`. Should we include that one as well and refer to it here?